mysql-test/suite/innodb/t/percona_force_encryption.test

```diff
@@ -0,0 +1,96 @@
+--source include/have_innodb.inc
```

Comment:
Since the feature goes through .FRM, its cold reading should be tested: add FLUSH TABLES, query the same tables again